setup.py

```diff
@@ -33,7 +33,7 @@ def get_version():
 
 
 setup(
-    name="pymmmc",
+    name="pymc_marketing",
```

Comment:
This is the name it'll get on pypi, no? So I guess same question.

Reply:
In pymc-experimental it seems to have the underscore? https://github.com/pymc-devs/pymc-experimental/blob/5a01d4fc5d58fc1abb52f13fcb5cd4e8b32292ba/setup.py#L22

Reply:
Yeah, that looks like solid evidence to me :D